packages/notebook-extension/src/index.ts

```diff
@@ -1491,7 +1491,7 @@ function populateMenus(app: JupyterLab, mainMenu: IMainMenu, tracker: INotebookT
   // Add a clearer to the edit menu
   mainMenu.editMenu.clearers.add({
     tracker,
-    noun: 'Cell Outputs',
+    noun: 'Cells\' Outputs',
```

Comment:
Are you going to change the rest of the cell output things we talked about in this PR or a follow on one?

Reply:
I'll make all the changes in this PR, so let's wait on this (away from my laptop at the moment)

Reply:
Let me ask around the UI team about possessives in menus. It seems very odd to me to have an apostrophe in a menu label.

Reply:
Sounds good.

Reply:
I asked, and one feedback was `Clear Cells' Output` feels weird. I agree - it really feels odd to have the possessive apostrophe, so -1 on this change.

Reply:
This PR is missing the full discussion though. "Clear Cells' Output" is not grammatically correct. It is similar to "websites' URLs" - the second word also has to be plural when the first word is plural possessive. Thus, our recommendation was "Clear Cells' Outputs" which is more clear than "Clear Cell Outputs" - which misleadingly only refers to one cell.

Reply:
I also am thinking that the plural possessive looks weird. I think that, rather than `Clear Cells' Output`, I would advocate for `Clear Selected Outputs` and `Clear All Outputs`. This is both grammatically correct, and consistent with the ways that we are referring to the outputs in the expand/collapse sections of the `View` menu.

Reply:
@ian-r-rose great idea, let's go with that for now. I think it goes without saying that all outputs are part of cells so that is redundant.